### ISSUE TITLE
Admin Dashboard banner (at login and after login) which indicates which PostgreSQL DB is active on Heroku

### ISSUE THREAD

**Issue by enoren5** [enoren5/tarot_juicer#124]
Admin Dashboard banner (at login and after login) which indicates which PostgreSQL DB is active on Heroku.
Check out: `$ heroku pg:info --app tarot-prod` (copied below). 

The Django Admin Dashboard sign on page (as well as after having successfully logged in) needs a  banner to notify/indicate the user which DATABASE_URL is in use, as well as the Add-on name. 

For example, if `HEROKU_POSTGRESQL_NAVY_URL`/ ` postgresql-real-1` is promoted as active and in use, then Admin Dashboard needs to say that in the form of a banner

Here are all the PostgreSQL DB's currently available for my project:
```
$ heroku pg:info --app tarot-prod
=== DATABASE_URL, HEROKU_POSTGRESQL_SILVER_URL
Plan:                  Hobby-dev
Status:                Available
Connections:           0/20
PG Version:            12.7
Created:               2020-11-19 10:32 UTC
Data Size:             8.2 MB
Tables:                0
Rows:                  0/10000 (In compliance)
Fork/Follow:           Unsupported
Rollback:              Unsupported
Continuous Protection: Off
Add-on:                postgresql-lorem-1

=== HEROKU_POSTGRESQL_NAVY_URL
Plan:                  Hobby-dev
Status:                Available
Connections:           2/20
PG Version:            12.7
Created:               2021-01-09 20:43 UTC
Data Size:             9.5 MB
Tables:                21
Rows:                  113/10000 (In compliance)
Fork/Follow:           Unsupported
Rollback:              Unsupported
Continuous Protection: Off
Add-on:                postgresql-real-1

=== HEROKU_POSTGRESQL_PUCE_URL
Plan:                  Hobby-dev
Status:                Available
Connections:           0/20
PG Version:            13.3
Created:               2021-03-20 17:37 UTC
Data Size:             8.2 MB
Tables:                0
Rows:                  0/10000 (In compliance)
Fork/Follow:           Unsupported
Rollback:              Unsupported
Continuous Protection: Off
Add-on:                postgresql-animate-93816
```

**Comment by enoren5** [comment]
@UmarGit : PR's #127 and #128 have completed most of this Issue, however the full feature is not quite implemented. The banner needs to appear _after_ login but also _before_ (as described originally above). Would you be able to complete this feature?

**Comment by UmarGit** [comment]
> @UmarGit : PR's #127 and #128 have completed most of this Issue, however the full feature is not quite implemented. The banner needs to appear _after_ login but also _before_ (as described originally above). Would you be able to complete this feature?

Like when we visited the admin login page.

**Comment by enoren5** [comment]
Yes, please

**Comment by enoren5** [comment]
Hi @UmarGit! Both inside and outside the Admin Dashboard in staging and prod pipelines, Django is showing me: "Currently there is no database found". See here: https://imgur.com/a/BkY3XOA

I had a similar problem last time. You suggested using a key combination with "F5" which worked for you but not for me. The solution last time for me was deleting my saved cookies . That would ensure that Django shows the db name inside the Admin Dashboard. But deleting cookies this time doesn't seem to resolve the issue. 

I'm wondering if this is an issue with your code or perhaps this might be an issue with the way I am handling it.

What would you recommend I try next?

**Comment by enoren5** [comment]
I just merged your changes into master and pushed to the staging pipeline. The Database ID is still not showing. See here: https://i.imgur.com/Ie4JI1D.jpg 

Also, @UmarGit: Make sure you are making changes to the latest version of master. It seems to me that the current GitHub repo's version of master is way more commits ahead of yours.

**Comment by enoren5** [comment]
@UmarGit: I merged your changes into the master branch and pushed to the staging pipeline. When logging into the Django Admin Dashboard, I get this: https://i.imgur.com/DFRsS9V.jpg

"Currently you are running app on localhost"

The db ID is still not showing. I tried deleting my cookies. Still not corrected.

**Comment by enoren5** [comment]
Implemented in #134. Thanks, @UmarGit!
